astropy/cosmology/core.py

```diff
@@ -82,8 +82,8 @@ class Cosmology(metaclass=abc.ABCMeta):
     write = UnifiedReadWriteMethod(CosmologyWrite)
 
     # Parameters
-    __parameters__ = ()
-    __all_parameters__ = ()
+    __parameters__: Tuple[str]  = ()
+    __all_parameters__: Tuple[str] = ()
```

Comment:
Shouldn't the types be the following?
```suggestion
    __parameters__: Tuple[str, ...]  = ()
    __all_parameters__: Tuple[str, ...] = ()
```

Reply:
Hmm, I was going to comment about ensuring PRs just do one thing... Maybe typing changes could be separate - at least, I shouldn't be reviewing them!

Reply:
@eerovaher, Yes! I used VSCode's auto-fix, but it clearly doesn't have perfect introspection.